buildkite/src/Jobs/ArchiveNode.dhall

```diff
@@ -0,0 +1,49 @@
+let Prelude = ../External/Prelude.dhall
+let Cmd = ../Lib/Cmds.dhall
+let S = ../Lib/SelectFiles.dhall
+let Pipeline = ../Pipeline/Dsl.dhall
+let JobSpec = ../Pipeline/JobSpec.dhall
+let Command = ../Command/Base.dhall
+let OpamInit = ../Command/OpamInit.dhall
+let Docker = ../Command/Docker/Type.dhall
+let Size = ../Command/Size.dhall
+in
+
+let user = "admin"
+let password = "codarules"
+let db = "archiver"
```

Comment:
these variables are dead code now, right? Can we remove them?

Reply:
I am using these variables when I set up the database. I am passing these to the database script.